template/src/styles/theme/index.ts

```diff
@@ -1,20 +1,39 @@
-import type { TRule } from "fela";
+import { createStitches } from '@stitches/react';
 
-export const theme = {
-  colors: {},
-  fonts: {},
-  sizes: {},
-  spacing: {},
-  boxShadows: {},
-  zIndex: {},
-} as const;
-
-export type Theme = typeof theme;
-
-export type ThemeProps = { theme: Theme };
-
-export type TRuleWithTheme<Props = {}> = TRule<ThemeProps & Props>;
-
-export type RulesExtend<TExtandalbeRules, TProps = {}> = Partial<
-  Record<keyof TExtandalbeRules, TRuleWithTheme<TProps>>
->;
+export const { styled, css, globalCss, keyframes, getCssText, theme, createTheme, config } = createStitches({
+    theme: {
```

Comment:
A lot of things here are project specific, I would delete it, you would do it when bootstraping a new project anyways.

Reply:
oks 👍